### PR TITLE
[DASH-2332] File name validation doesn't fully work

### PR DESCRIPTION
**Pull request by tomsonpl**
screen shot 2016-10-28 at 11 01 37

Apparently, the validation worked fine, and the files I uploaded were "correct". So I decided to use truncate to shorten these files, maximum of 64 letters, do you think it's enough?

### CODE REVIEW

**Comment by Flixow** [comment]
What about give possibility to vertical scroll file name instead of truncate it? Or second way validate all names on upload.

**Comment by tomsonpl** [comment]
@Flixow I think that this case (that I had) will happen very rarely, is there a point in making a vertical scroll just for one? it may look weird.    Validate name or backend or frontend? How do you want it to be validated? For length? 
What I was thinking is making truncate onDefault, and then onclick showing more of a file name. What do you think?

**Comment by Flixow** [comment]
Yes, I thought about validate file name based on length. Max 120 characters for example.
Looking for your idea, how do you want inform user that he should click for showing full file name?

**Comment by tomsonpl** [comment]
And what if a user would like to upload a file longer than 120 chars? 
I would leave it truncated, personally if I would see a truncated text, I would just click it, even without an info - to check if it extends. However, we can ofc add a hint - click to see the full name or something. But then I would have to somehow take care of this again, maybe wrap text would do the trick

**Comment by patrykkopycinski** [comment]
if backend allows to upload files with long names leave it as it is. in case of truncating long names use css to truncate it

File: src/apps/Hosting/HostingFilesListItem.js [hunk 1]
```diff
@@ -65,7 +67,11 @@ const HostingFileListItem = Radium(({ item, onFolderEnter, onIconClick, showDele
       );
     }
 
-    return fileName;
+    return (
+      <div style={styles.truncate} >
```
Comment (Flixow) [inline src/apps/Hosting/HostingFilesListItem.js]: You can use Truncate Component, can't you?

Comment (tomsonpl) [inline src/apps/Hosting/HostingFilesListItem.js]: it doesn't change a thing. I still have to set width manually.

Comment (Flixow) [inline src/apps/Hosting/HostingFilesListItem.js]: So, on my mind this Truncate Component should be improved?

Comment (tomsonpl) [inline src/apps/Hosting/HostingFilesListItem.js]: Sounds like a great idea, lets create a task for that :P

**Comment by tomsonpl** [comment]
Can anybody please tell me what exactly is wrong with these tests? Every freaking time it shows different errors ;/

**Comment by gh3r** [comment]
@tomsonpl I will check it

**Pull request merged**